core/services/relay/evm/evm.go

```diff
@@ -524,13 +525,35 @@ func (r *Relayer) NewCCIPCommitProvider(ctx context.Context, rargs commontypes.R
 		return nil, err
 	}
 	subjectID := chainToUUID(configWatcher.chain.ID())
-	contractTransmitter, err := newOnChainContractTransmitter(ctx, r.lggr, rargs, r.evmKeystore, configWatcher, configTransmitterOpts{
+
+	var contractTransmitter ContractTransmitter
+	evmContractTransmitter, err := newOnChainContractTransmitter(ctx, r.lggr, rargs, r.evmKeystore, configWatcher, configTransmitterOpts{
 		subjectID: &subjectID,
 	}, OCR2AggregatorTransmissionContractABI, WithReportToEthMetadata(fn), WithRetention(0))
 	if err != nil {
 		return nil, err
 	}
 
+	if configWatcher.chain.Config().EVM().ChainType() == chaintype.ChainTron {
+		tronTransmitter, err := NewTronContractTransmitter(ctx, r.lggr, TronContractTransmitterOpts{
+			EVMTransmitter:        evmContractTransmitter,
+			Keystore:              r.evmKeystore,
+			ConfigWatcher:         configWatcher,
+			ConfigTransmitterOpts: configTransmitterOpts{subjectID: &subjectID},
+		}, WithReportToEthMetadata(fn), WithRetention(0))
+		if err != nil {
+			return nil, err
+		}
+
+		if err := tronTransmitter.Start(ctx); err != nil {
```

Comment:
We (1) start it in the constructor, (2) here, (3) on plugin start - we need a clean lifecycle for this and every other component.